src/includes/class-alma-wc-settings.php

```diff
@@ -60,22 +60,22 @@ class Alma_WC_Settings {
 	 * @return array
 	 */
 	public static function default_settings() {
-		$payment_methods_description = __( 'Fast and secure payment by credit card', 'alma-woocommerce-gateway' );
+		$payment_methods_description = 'Fast and secure payment by credit card';
 		return array(
 			'enabled'                               => 'yes',
 			'selected_fee_plan'                     => self::DEFAULT_FEE_PLAN,
 			'enabled_general_3_0_0'                 => 'yes',
-			'title_payment_method_pnx'              => __( 'Pay in installments with Alma', 'alma-woocommerce-gateway' ),
+			'title_payment_method_pnx'              => 'Pay in installments with Alma',
```

Comment:
We have to keep translations called somewhere, else, the translations keys will disappear when we will parse the code with poedit

Reply:
True, Poedit won't find them anymore !
But translate those strings in the setting class would have for effet to lose the key value and make translations in other languages impossible. 
Open subject...